code/game/objects/items/stacks/stack.dm

```diff
@@ -430,7 +430,7 @@
 /obj/item/stack/proc/can_merge(obj/item/stack/check)
 	if(!istype(check, merge_type))
 		return FALSE
-	if(mats_per_unit != check.mats_per_unit)
+	if(mats_per_unit ~! check.mats_per_unit)
```

Comment:
Comment why this is being done

Reply:
in the code, or just why i changed that?

Reply:
next to this if() check, describe why we're doing it. Cause you're right it's a little bit hacky, and the operator is a bit rare, so it's best to make sure people know what/why we're doing